x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -437,7 +437,7 @@ ${JSON.stringify(cypressConfigFile, null, 2)}
 
               const productTypes = isOpen
                 ? getProductTypes(tier, endpointAddon, cloudAddon)
-                : (parseTestFileConfig(filePath).productTypes as ProductType[]);
+                : (parseTestFileConfig(filePath).ftrConfig?.productTypes as ProductType[]);
```

Comment:
Looks like `ftrConfig` is always present. The `productTypes` could be `undefined` which is why it is type casted here.

```suggestion
                : (parseTestFileConfig(filePath).ftrConfig.productTypes as ProductType[]);
```